src/scripts/OutSystems/OSUI/Patterns/TimePickerAPI.ts

```diff
@@ -171,6 +171,26 @@ namespace OutSystems.OSUI.Patterns.TimePickerAPI {
 		return _timePickerItem;
 	}
 
+	/**
+	 * Function that will be triggered everytime there is a render at TimePicker
+	 *
+	 * @export
+	 * @param {string} timePickerId
+	 * @return {*}  {string}
+	 */
+	export function OnRender(timePickerId: string): string {
+		const result = OutSystems.OSUI.Utils.CreateApiResponse({
+			errorCode: ErrorCodes.TimePicker.FailOnRender,
+			callback: () => {
+				const _timePickerItem = GetTimePickerItemById(timePickerId);
```

Comment:
The other APIs are using `this.GetTimePickerItemById`. Shouldn't this one be like this too?

Reply:
nice catch 💪